samcli/commands/_utils/custom_options/option_help.py

```diff
@@ -0,0 +1,28 @@
+"""
+Custom click option for customizing help text.
+"""
+
+import click
+
+from samcli.lib.utils.colors import Colored
+
+
+class RequiredOptionColorTextOption(click.Option):
```

Comment:
Is there a way to can do this on `click` directly? Which would allow us to not have to attach it to individual options?

Reply:
Not with click directly, Yeah I did think about ordering! Terraform cli has a nice way of doing this, showcasing the core options first.

In terms of colors, other ideas are to just to think about bold text instead.